tests/integration/targets/azure_rm_virtualmachinescaleset/tasks/main.yml

```
@@ -27,7 +27,9 @@
   azure_rm_loadbalancer:
     resource_group: "{{ resource_group }}"
     name: testLB
-    public_ip_address_name: testPublicIP
+    frontend_ip_configurations:
+      - name: frontendipconf0
+        public_ip_address: testPublicIP
```

Add backend_address_pools.  Before the deletion, backend_address_pools has a default value if it is not configured
```suggestion
        public_ip_address: testPublicIP
      backend_address_pools:
        - name: backendaddrp0
```